src/Features/Clustered.cpp

```diff
@@ -57,35 +57,17 @@ void Clustered::UpdateLights()
 		color.z = dimmer * niLight->GetLightRuntimeData().diffuse.blue;
 		light.color = XMLoadFloat3(&color);
 
-
-
-		if (REL::Module::IsVR()) {
-			DirectX::XMFLOAT3 position{};
-
-			worldPos = worldPos - state->GetVRRuntimeData().posAdjust.getEye();
-
-			position.x = worldPos.x;
-			position.y = worldPos.y;
-			position.z = worldPos.z;
-			light.positionWS = XMLoadFloat3(&position);
-
-			worldPos = niLight->world.translate;
-
-			worldPos = worldPos - state->GetVRRuntimeData().posAdjust.getEye(1);
-
-			position.x = worldPos.x;
-			position.y = worldPos.y;
-			position.z = worldPos.z;
-			light.positionWS2 = XMLoadFloat3(&position);
-		} else {
+		for (int eyeIndex = 0; eyeIndex < (!REL::Module::IsVR() ? 1 : 2); eyeIndex++) {
 			DirectX::XMFLOAT3 position{};
-
-			worldPos = worldPos - state->GetVRRuntimeData().posAdjust.getEye();
```

Comment:
@doodlum note this flat code was wrong since it's use `GetVRRuntimeData()`. Even though I fixed it, there appears to be an alignment issue. Perhaps the attempt to combine the positionWS and positionWS2 into an array breaks at alignas16? The Flat code is significantly different because it does try to update `positionVS` (which should be `positionWS[1]` now.

See 
https://github.com/doodlum/skyrim-community-shaders/blob/main/src/Features/Clustered.cpp#L76-L80. Even writing to the `positionWS[1]` with the older `XMVector3TransformCoord(light.positionWS, state->GetVRRuntimeData().cameraData.getEye().viewMat);` didn't seem to solve the issue I'm seeing.